pkg/ingester/ingester.go

```diff
@@ -173,6 +174,7 @@ type Ingester struct {
 	wal WAL
 
 	chunkFilter storage.RequestChunkFilterer
+	labelFilter listutil.LabelValueFilterer
```

Comment:
This `LabelValueFilterer` interface is only used here, so I'd like that we defined it here only.

Reply:
Moved.